src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -214,14 +215,14 @@ std::map<std::string, std::string> Plugin::ConvertPerfHintsToConfig(
                                : plugin_config.perfHintsConfig.ovPerfHint;
         //checking streams (to avoid overriding what user might explicitly set in the incoming config or previously via SetConfig)
         const auto streams = config.find(PluginConfigParams::KEY_GPU_THROUGHPUT_STREAMS) == config.end() &&
-                             config.find(ov::streams::num.name()) == config.end();
+                             config.find(ov::num_streams.name()) == config.end();
         if (streams && !streamsSet) {
             if (mode_name == CONFIG_VALUE(LATENCY)) {
                 config[PluginConfigParams::KEY_GPU_THROUGHPUT_STREAMS] = std::to_string(1);
-                config[ov::streams::num.name()] = std::to_string(1);
+                config[ov::num_streams.name()] = std::to_string(1);
             } else if (mode_name == CONFIG_VALUE(THROUGHPUT)) {
                 config[PluginConfigParams::KEY_GPU_THROUGHPUT_STREAMS] = CONFIG_VALUE(GPU_THROUGHPUT_AUTO);
-                config[ov::streams::num.name()] = std::to_string(ov::streams::AUTO);
+                config[ov::num_streams.name()] = ov::Any::make<ov::NumStreams>(ov::NumStreams::AUTO).as<std::string>();
```

Comment:
why don't we use `util::to_string(ov::NumStreams(ov::NumStreams::AUTO))` here?

Reply:
the code was written when there was no `ov::to_string`. Missed to replace later.